docker/ci/ubuntu/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM ubuntu:22.04
 ARG DOCKER_BUILD_ROOT=.
 # The binary name of GreptimeDB executable.
 # Defaults to "greptime", but sometimes in other projects it might be different.
-ARG TARGET_BIN=greptime
+ENV TARGET_BIN=greptime
```

Comment:
I don’t think removing ARG is a good solution. Using ARG instruction can let users pass the variables from Docker easily. You should still keep ARG and add another instruction for injecting env from the ARG variable.